srcs/python/kungfu/tensorflow/policy/scaling.py

```diff
@@ -0,0 +1,85 @@
+import tensorflow as tf
```

Comment:
what's the difference between this file and `scaling_policy.py`

Reply:
`scaling.py` is a working hook while `scaling_policy.py` is my working file to adjust to the policy guidline